src/plugins/WebOfTrust/l10n/lang_ru.l10n

```diff
@@ -74,16 +74,6 @@ KnownIdentitiesPage.KnownIdentities.Table.NicknameNotDownloadedYet=Еще не 
 KnownIdentitiesPage.KnownIdentities.Table.NoScore=Нет
 KnownIdentitiesPage.KnownIdentities.Table.UpdateTrustButton=Обновить
 KnownIdentitiesPage.SetTrust.Failed=Ошибка при установке доверия
-MyIdentityPage.OwnIdentities.OwnIdentityTable.DeleteButton=Удалить
-MyIdentityPage.OwnIdentities.OwnIdentityTable.EditButton=Редактировать
-MyIdentityPage.OwnIdentities.OwnIdentityTableHeader.LastChange=Последнее изменение
-MyIdentityPage.OwnIdentities.OwnIdentityTableHeader.LastInsert=Последняя вставка
-MyIdentityPage.OwnIdentities.OwnIdentityTableHeader.Manage=Управлять
-MyIdentityPage.OwnIdentities.OwnIdentityTableHeader.Name=Имя
-MyIdentityPage.OwnIdentities.OwnIdentityTableHeader.PublishesTrustlist=Публикует лист доверия
-MyIdentityPage.OwnIdentities.OwnIdentityTableHeader.Trusters=Доверители
-MyIdentityPage.OwnIdentities.OwnIdentityTable.IntroduceButton=Представиться
-StatisticsPage.SummaryBox.FetchProgress=Общее количество всех изменений: ${editionCount}
```

Comment:
I cannot speak Russian, but can say that its strange why so many of those disappeared?

I would have looked into the history of those strings on Transifex, but I cannot since i am only on the German translation team.

Not a merge blocker though, as with the other stuff just telling you so you can fix bugs in your scripts.